ale/drivers/dawn_drivers.py

```diff
@@ -157,3 +158,21 @@ def focal2pixel_lines(self):
         # Microns to mm
         pixel_size = spice.gdpool('INS{}_PIXEL_SIZE'.format(self.ikid), 0, 1)[0] * 0.001
         return [0.0, 0.0, 1/pixel_size]
+
+    def detector_start_line(self):
+        return 1
+
+    def detector_start_sample(self):
+        return 1
+
+    @property
+    def sensor_model_version(self):
+        """
+        Returns instrument model version
+
+        Returns
+        -------
+        : int
+          model version
+        """
+        return 2
```

Comment:
Was this a placeholder as well?

Reply:
No, I got this value from the dawnfc camera.plugin file in the isis repo